### PR TITLE
[KO] Update index to 2.0

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: ko/index.md [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-+ 높은 장애 복원력
-+ 함수형 프로그래밍
-+ 확장할 수 있는 언어
+Elixir School의 목표는 Elixir 프로그래밍 언어를 배우려는 사람들에게 최고의 선택지가 되는 것입니다.
+
+노련한 숙련자던 초심자던 관계없이 레슨과 보조 자료에서 필요한 것을 찾을 수 있습니다.
+
+자원봉사자들의 노력으로 Elixir School은 많은 언어로 번역되어 있습니다. 다음은 그중 일부입니다. [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], [ภาษาไทย][th].
+
+당신도 [elixirschool/elixirschool](https://github.com/elixirschool/elixirschool)에 참여하여 Elixir School을 더욱 멋지게 만드실 수 있습니다!
```
Comment (devleoper) [inline ko/index.md]:
```suggestion
[elixirschool/elixirschool](https://github.com/elixirschool/elixirschool)에 참여하여 Elixir School을 더욱 멋지게 만드실 수 있습니다!
```

File: ko/index.md [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
-
-+ 규모 가변적
-+ 높은 장애 복원력
-+ 함수형 프로그래밍
-+ 확장할 수 있는 언어
+Elixir School의 목표는 Elixir 프로그래밍 언어를 배우려는 사람들에게 최고의 선택지가 되는 것입니다.
+
+노련한 숙련자던 초심자던 관계없이 레슨과 보조 자료에서 필요한 것을 찾을 수 있습니다.
+
+자원봉사자들의 노력으로 Elixir School은 많은 언어로 번역되어 있습니다. 다음은 그중 일부입니다. [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], [ภาษาไทย][th].
```
Comment (devleoper) [inline ko/index.md]:
```suggestion
자원봉사자들의 노력으로 Elixir School은 [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], [ภาษาไทย][th] 등 많은 언어로 번역되어 있습니다.
```

**Comment by marocchino** [comment]
I realized I don't have merge permission of Korean. 😅  Would you check it, please?

**Comment by yosangwon** [comment]
Okay!

**Pull request merged**